mappings/net/minecraft/nbt/NbtElement.mapping

```diff
@@ -59,20 +59,24 @@ CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
 		COMMENT @see NbtCompound#contains(String, int)
 	FIELD field_33264 MAX_DEPTH I
 	METHOD method_10707 copy ()Lnet/minecraft/class_2520;
-		COMMENT Copies this NBT element.
-		COMMENT
-		COMMENT @return the copied element
+		COMMENT {@return the copied NBT element}
```

Comment:
```suggestion
		COMMENT {@return a copy of this NBT element}
```
Also, does this copy always perform a copy, or does it return the element itself if it's immutable?

Reply:
good point (it returns itself, yes)

Reply:
Maybe like ``
```suggestion
		COMMENT {@return an NBT element of equal value that won't change with this element}
```